### ISSUE TITLE
退出代码-1073741819 (0xC0000005)

### ISSUE THREAD

**Issue by standyyyy** [PaddlePaddle/PaddleSpeech#1711]
代码一运行过会就停止了，尝试过网络上那些更改h5py版本办法 ，但是还是不行  。
![image](https://user-images.githubusercontent.com/39285732/163667723-83167c8c-9f74-4387-8659-ef3c1ab92f8f.png)

代码：
asr_executor = ASRExecutor()
text = asr_executor(
    model='conformer_wenetspeech',
    lang='zh',
    sample_rate=16000,
    config=None,  # Set `config` and `ckpt_path` to None to use pretrained model.
    ckpt_path=None,
    audio_file='F:\github\py_speech_seg-master/1234.wav',
    force_yes=False,
    device=paddle.get_device())
print('ASR Result: \n{}'.format(text))
用的是windows 。

**Comment by zh794390558** [comment]
demos 里提供的音频能正常运行吗？

**Comment by standyyyy** [comment]
是可以正常播放的 。

**Comment by zh794390558** [comment]
asr可以正常运行？那就check下是否是输入音频的问题。

**Comment by standyyyy** [comment]
抱歉 ，理解错了，我以为你是资讯我所使用的音频是否可以正常播放 ，Demo里面的zh.wav也无法正常使用 。

以下是全部报错情况 。
`D:\Anaconda3\lib\site-packages\numpy\_distributor_init.py:32: UserWarning: loaded more than 1 DLL from .libs:
D:\Anaconda3\lib\site-packages\numpy\.libs\libopenblas.QVLO2T66WEPI7JZ63PS3HMOHFEY472BC.gfortran-win_amd64.dll
D:\Anaconda3\lib\site-packages\numpy\.libs\libopenblas.XWYDX2IKJW2NMTWSFYNGFUWKQU3LYTCZ.gfortran-win_amd64.dll
  stacklevel=1)
D:\Anaconda3\lib\site-packages\socks.py:58: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Callable
D:\Anaconda3\lib\site-packages\numba\types\__init__.py:110: DeprecationWarning: `np.long` is a deprecated alias for `np.compat.long`. To silence this warning, use `np.compat.long` by itself. In the likely event your code does not need to work on Python 2 you can use the builtin `int` for which `np.compat.long` is itself an alias. Doing this will not modify any behaviour and is safe. When replacing `np.long`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  long_ = _make_signed(np.long)
D:\Anaconda3\lib\site-packages\numba\types\__init__.py:111: DeprecationWarning: `np.long` is a deprecated alias for `np.compat.long`. To silence this warning, use `np.compat.long` by itself. In the likely event your code does not need to work on Python 2 you can use the builtin `int` for which `np.compat.long` is itself an alias. Doing this will not modify any behaviour and is safe. When replacing `np.long`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  ulong = _make_unsigned(np.long)
C:\Users\Administrator\AppData\Roaming\Python\Python37\site-packages\librosa\core\constantq.py:1059: DeprecationWarning: `np.complex` is a deprecated alias for the builtin `complex`. To silence this warning, use `complex` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.complex128` here.
Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
  dtype=np.complex,
D:\Anaconda3\lib\site-packages\nltk\decorators.py:70: DeprecationWarning: `formatargspec` is deprecated since Python 3.5. Use `signature` and the `Signature` object directly
  formatvalue=lambda value: "")[1:-1]
D:\Anaconda3\lib\site-packages\matplotlib\__init__.py:107: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import MutableMapping
D:\Anaconda3\lib\site-packages\matplotlib\rcsetup.py:20: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Iterable, Mapping
D:\Anaconda3\lib\site-packages\matplotlib\colors.py:53: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Sized
D:\Anaconda3\lib\site-packages\matplotlib\__init__.py:179: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  a = distutils.version.LooseVersion(a)
D:\Anaconda3\lib\site-packages\matplotlib\__init__.py:180: DeprecationWarning: distutils Version classes are deprecated. Use packaging.version instead.
  b = distutils.version.LooseVersion(b)
[2022-04-18 15:32:33,559] [    INFO] - checking the audio file format......
[2022-04-18 15:32:33,560] [    INFO] - The sample rate is 16000
[2022-04-18 15:32:33,560] [    INFO] - The audio file format is right
[2022-04-18 15:32:33,575] [    INFO] - File C:\Users\Administrator\.paddlespeech\models\conformer_wenetspeech-zh-16k\asr1_conformer_wenetspeech_ckpt_0.1.1.model.tar.gz md5 checking...
[2022-04-18 15:32:37,946] [    INFO] - Use pretrained model stored in: C:\Users\Administrator\.paddlespeech\models\conformer_wenetspeech-zh-16k\asr1_conformer_wenetspeech_ckpt_0.1.1.model.tar
[2022-04-18 15:32:37,946] [    INFO] - C:\Users\Administrator\.paddlespeech\models\conformer_wenetspeech-zh-16k\asr1_conformer_wenetspeech_ckpt_0.1.1.model.tar
[2022-04-18 15:32:37,946] [    INFO] - C:\Users\Administrator\.paddlespeech\models\conformer_wenetspeech-zh-16k\asr1_conformer_wenetspeech_ckpt_0.1.1.model.tar\model.yaml
[2022-04-18 15:32:37,946] [    INFO] - C:\Users\Administrator\.paddlespeech\models\conformer_wenetspeech-zh-16k\asr1_conformer_wenetspeech_ckpt_0.1.1.model.tar\exp/conformer/checkpoints/wenetspeech.pdparams
2022-04-18 15:32:38.259 | INFO     | paddlespeech.s2t.frontend.featurizer.text_featurizer:_load_vocabulary_from_file:229 - BLANK id: 0
2022-04-18 15:32:38.259 | INFO     | paddlespeech.s2t.frontend.featurizer.text_featurizer:_load_vocabulary_from_file:230 - UNK id: 1
2022-04-18 15:32:38.260 | INFO     | paddlespeech.s2t.frontend.featurizer.text_featurizer:_load_vocabulary_from_file:231 - EOS id: 5536
2022-04-18 15:32:38.260 | INFO     | paddlespeech.s2t.frontend.featurizer.text_featurizer:_load_vocabulary_from_file:232 - SOS id: 5536
2022-04-18 15:32:38.260 | INFO     | paddlespeech.s2t.frontend.featurizer.text_featurizer:_load_vocabulary_from_file:233 - SPACE id: -1
2022-04-18 15:32:38.260 | INFO     | paddlespeech.s2t.frontend.featurizer.text_featurizer:_load_vocabulary_from_file:234 - MASKCTC id: -1
WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.
WARNING: Ignoring invalid distribution -ip (d:\anaconda3\lib\site-packages)
WARNING: Ignoring invalid distribution -ip (d:\anaconda3\lib\site-packages)
Looking in indexes: http://mirrors.aliyun.com/pypi/simple
ERROR: Could not find a version that satisfies the requirement paddlespeech_ctcdecoders (from versions: none)
ERROR: No matching distribution found for paddlespeech_ctcdecoders
WARNING: Ignoring invalid distribution -ip (d:\anaconda3\lib\site-packages)
WARNING: Ignoring invalid distribution -ip (d:\anaconda3\lib\site-packages)
2022-04-18 15:32:40.374 | INFO     | paddlespeech.s2t.modules.ctc:<module>:43 - paddlespeech_ctcdecoders not installed!
2022-04-18 15:32:40.414 | INFO     | paddlespeech.s2t.models.u2.u2:_init_from_config:829 - U2 Encoder type: conformer
2022-04-18 15:32:45.825 | INFO     | paddlespeech.s2t.modules.loss:__init__:41 - CTCLoss Loss reduction: sum, div-bs: True
2022-04-18 15:32:45.825 | INFO     | paddlespeech.s2t.modules.loss:__init__:42 - CTCLoss Grad Norm Type: None
2022-04-18 15:32:45.826 | INFO     | paddlespeech.s2t.modules.loss:__init__:73 - CTCLoss() kwargs:{'norm_by_times': False}, not support: {'norm_by_batchsize': False, 'norm_by_total_logits_len': False}
[2022-04-18 15:32:48,447] [    INFO] - Preprocess audio_file:F:\Aawant\jinzhuan\zh.wav
[2022-04-18 15:32:48,447] [    INFO] - get the preprocess conf
[2022-04-18 15:32:49,323] [    INFO] - read the audio file
[2022-04-18 15:32:49,323] [    INFO] - audio shape: (79949,)
[2022-04-18 15:32:49,854] [    INFO] - audio feat shape: [1, 498, 80]

进程已结束,退出代码-1073741819 (0xC0000005)
`

**Comment by Jackwaterveg** [comment]
请问 直接 paddlespeech asr --input xxx.wav 可以运行吗？ @standyyyy

**Comment by standyyyy** [comment]
![image](https://user-images.githubusercontent.com/39285732/163947825-36aa968f-604f-48da-b3e1-85575eb78748.png)
尝试了一下，是可以的，库版本冲突的问题估计 。
